src/js/_enqueues/lib/comment-reply.js

```diff
@@ -173,6 +175,8 @@ window.addComment = ( function( window ) {
 		// Move the respond form back in place of the temporary element.
 		temporaryElement.parentNode.replaceChild( respondElement ,temporaryElement );
 		cancelLink.style.display = 'none';
+		var replyHeading = document.getElementsByClassName( config.commentReplyTitleInnerClass )[0];
+		replyHeading.textContent = headingText;
```

Comment:
Same as above, seems a good idea to check if `replyHeading` exists before using it.